apps/drupal/includes/form.theme.inc

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/**
+ * @file
+ * Functions to support theming form layout in the Particle theme.
+ */
+
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Implements hook_theme_suggestions_HOOK_alter().
+ */
+function particle_theme_suggestions_form_alter(array &$suggestions, array $variables) {
+  $id = $variables['element']['#id'];
+  // Particle Theme for Octane's Views Search and Drupal's Built-In Search.
+  // This hook allows to apply two search block implementations to the same
+  // theme file. Drupal only template file located at
+  // /particle/apps/drupal/templates/form/form--particle-search.html.twig.
+  if ($id == 'views-exposed-form-search-page' || $id == 'search-block-form') {
+    $suggestions[] = 'form__particle_search';
```

Comment:
Don't think "particle" name should be used here.  Projects have their own theme name (renamed from Particle) so would prefer a non-specific name more related to function here.

Reply:
This is what's currently in Particle. I think we need to consider all default theme hooks as "this makes a demo-able version on install and is intended to be deleted."